resources/views/settings/account.blade.php

```diff
@@ -83,63 +83,72 @@
             </div>
         </div>
     </div>
-    @if(config('auth.2fa_enabled'))
-        @if (auth()->user()->subscribed('kanka') || auth()->user()->subscription('kanka')->cancelled() || $user->PasswordSecurity->google2fa_enable)
+    @if (config('google2fa.enabled'))
         <div class="box box-solid">
             <div class="box-header with-border">
                 <h3 class="box-title">
                     {{ __('settings.account.2fa.title') }}
                 </h3>
             </div>
-            <div class="box-body">
-                @if (auth()->user()->isSocialLogin())
-                    <p>{{ __('settings.account.2fa.social') }}</p>
-                @else
-                    <!-- google2fa -->
-                    <p>{{ __('settings.account.2fa.helper') }}</p>
-                    @if(!$user->PasswordSecurity && auth()->user()->subscribed('kanka') || auth()->user()->subscription('kanka')->cancelled())
-                        {{ __('settings.account.2fa.enable_instructions') }}
-                        </div>
-                    <form method="POST" action="{{ route('generate2faSecretCode') }}">
-                        @csrf
-                        <div class="box-footer text-right">
-                            <button type="submit" class="btn btn-primary" name="button">{{ __('settings.account.2fa.generate_qr') }}</button>
-                        </div>
-                    </form>
+            @if (!auth()->user()->subscribed('kanka') && !isset($user->PasswordSecurity->google2fa_enable))
```

Comment:
Referencing `P` instead of `p`. Can't use `subscribed()` as they skips everyone using patreon/paypal/giropay and sofort payment methods. Use `$user->isPatron()` instead here and in the rest of the code.